src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -12,21 +16,41 @@ import { useExternalEnrollmentFailureReason, useMinimalCourseMetadata } from '..
 import ErrorPageContent from '../../executive-education-2u/components/ErrorPageContent';
 
 const ExternalCourseEnrollment = () => {
+  const config = getConfig();
   const history = useHistory();
   const {
     state: {
       activeCourseRun,
       courseEntitlementProductSku,
     },
     userSubsidyApplicableToCourse,
+    formSubmissionError,
   } = useContext(CourseContext);
+  const {
+    enterpriseConfig: { authOrgId },
+  } = useContext(AppContext);
+
   const courseMetadata = useMinimalCourseMetadata();
 
+  const externalDashboardQueryParams = new URLSearchParams({
+    org_id: authOrgId,
+  });
```

Comment:
We likely only should pass an `org_id` query parameter if the `EnterpriseCustomer` has one configured. Perhaps something like the following:

```jsx
const externalDashboardQueryParams = new URLSearchParams();
if (authOrgId) {
  externalDashboardQueryParams.set('org_id', authOrgId);
}

let externalDashboardUrl = config.GETSMARTER_LEARNER_DASHBOARD_URL;
if (externalDashboardQueryParams.size > 0) {
  externalDashboardUrl += `?${externalDashboardQueryParams.toString()}`;
}
```